sys/ssp/ssp.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_ssp Stack Smashing Protector
+ * @ingroup     sys
+ * @brief       Stack Smashing protector
+ *
+ * This module implements necessary helper functions that enable RIOT to make
+ * use of GCC's stack smashing protector (SSP).
+ *
+ * See http://wiki.osdev.org/Stack_Smashing_Protector for a more detailed
+ * description.
+ *
```

Comment:
Move to `sys/doc.txt` to achieve visibility. `.c` files are not included in Doxygen.